.eslintrc.json

```diff
@@ -18,7 +18,8 @@
     "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
     "react/react-in-jsx-scope": "off",
     "import/no-unresolved": "off",
-    "no-shadow": "off"
+    "no-shadow": "off",
+    "object-shorthand": "off"
   },
```

Comment:
- [ ] Please kindly remove this configuration option that you newly added to the linter as opposed the [given file](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json). When "object-shorthand" is set to "off" in a linting configuration, it likely means that the linting tool will not enforce or suggest the use of object shorthand syntax. ℹ️